packages/javascript-sdk/src/session-manager/index.ts

```diff
@@ -25,13 +25,12 @@ abstract class SessionManager {
    * Ends the current session.
    */
   public static async logout(options?: ConfigOptions): Promise<Response> {
-    const { middleware, realmPath, serverConfig } = Config.get(options);
+    const { middleware, platformHeader, realmPath, serverConfig } = Config.get(options);
```

Comment:
don't we need to add this to the setDefaults function? If this isn't passed in wouldn't this be undefined? I believe as it stands theres no real issue in terms of falsey checks, but i'm curious if it would just be more type aligned to default it to false if not opted in?

Reply:
Hmm, okay. Let me think on this for a minute.

Reply:
Okay, addressed.